book.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
GitBook is a nice feature, 
Do we still need to present the step of the nightclass with slides ? (slides could be kept for introduction 💡 )
We could update those descriptions with git tag we will create ? 🏷

Reply:
Do you think this exercises rules are enought ?

Reply:
It could be enough if we livecode thos exercices.